src/theory/arith/kinds

```diff
@@ -94,40 +94,40 @@ typerule POW ::CVC4::theory::arith::ArithOperatorTypeRule
 
 typerule CONST_RATIONAL ::CVC4::theory::arith::ArithConstantTypeRule
 
-typerule LT ::CVC4::theory::arith::ArithPredicateTypeRule
-typerule LEQ ::CVC4::theory::arith::ArithPredicateTypeRule
-typerule GT ::CVC4::theory::arith::ArithPredicateTypeRule
-typerule GEQ ::CVC4::theory::arith::ArithPredicateTypeRule
+typerule LT "SimpleTypeRule<RBool, AReal, AReal>"
```

Comment:
It's a bit counter-intuitive to use `<RBool, AReal, AReal>` instead of `<AReal, AReal, RBool>`, but I guess there is no other way with the optional arguments.

Reply:
Yeah, I am not sure if there is a better way, unfortunately. Any ideas?